local-cluster/tests/local_cluster.rs

```diff
@@ -4762,3 +4762,121 @@ fn test_duplicate_with_pruned_ancestor() {
         "test_duplicate_with_pruned_ancestor",
     );
 }
+
+/// Test fastboot to ensure a node can boot from local state
```

Comment:
I think this test aims to and successfully shows that a node can create snapshot from fastboot and then restart from the snapshot archive generated after that.

It might be out of the scope for the intent of this test, but I'm not sure it demonstrates that's done correctly? Should the test have other nodes on the cluster so that we can demonstrate that the restarted node actually gets the **correct** hash after the restarts?

Reply:
> Maybe checking snapshot hashes is sufficient?

That sounds good to me